src/components/VerticalNav/VerticalNav.js

```diff
@@ -4,14 +4,20 @@ import cx from 'classnames';
 import { ListGroup } from '../ListGroup';
 import VerticalNavItem from './VerticalNavItem';
 import VerticalNavMasthead from './VerticalNavMasthead';
-import { bindMethods, filterChildren, findChild } from '../../common/helpers';
+import {
+  bindMethods,
```

Comment:
open question here - would anyone be opposed to exposing/consuming all of our helpers like `bindMethods` and the others you've added in this PR from `src/index.js`? It would be nice to use these downstream as well I am thinking, but will see how others feel...